content/api/commands/intercept.md

```diff
@@ -922,42 +839,15 @@ A `StaticResponse` represents a stubbed response to an HTTP request. You can sup
 
 The following properties are available on `StaticResponse`. All properties are optional:
 
-```ts
-{
-  /**
-   * Serve a fixture as the response body.
-   */
-  fixture?: string
-  /**
-   * Serve a static string/JSON object as the response body.
-   */
-  body?: string | object | object[]
-  /**
-   * HTTP headers to accompany the response.
-   * @default {}
-   */
-  headers?: { [key: string]: string }
-  /**
-   * The HTTP status code to send.
-   * @default 200
-   */
-  statusCode?: number
-  /**
-   * If 'forceNetworkError' is truthy, Cypress will destroy the browser connection
-   * and send no response. Useful for simulating a server that is not reachable.
-   * Must not be set in combination with other options.
-   */
-  forceNetworkError?: boolean
-  /**
-   * Milliseconds to delay before the response is sent.
-   */
-  delay?: number
-  /**
-   * Kilobits per second to send 'body'.
-   */
-  throttleKbps?: number
-}
-```
+| Option            | Default | Description                                            |
+|-------------------|---------|--------------------------------------------------------|
+| fixture           | null    | Serve a fixture as the HTTP response body              |
+| body              | null    | Serve a static string/JSON object as the response body |
+| headers           | {}      | HTTP response headers                                  |
+| statusCode        | 200     | HTTP response status code                              |
+| forceNetworkError | null    | Force an error by destroying the browser session       |
+| delay             | 0       | Minimum network latency/delay (milliseconds)           |
```

Comment:
In the snippet, we don't specify this default, but it still makes sense.

Reply:
There's latency in the network irrespective of this, so I wonder if we _add_ this delay to that.